### DISCUSSION TITLE
新的mac系统不支持微信助手

### DISCUSSION THREAD

**Discussion by ahuaiph** [MustangYM/WeChatExtension-ForMac#822] [Q&A]
更新了macos系统之后，就不能支持微信助手了，报错信息为找不到相应的依赖。微信重新装了，微信助手重新装了，都不行。
![image](https://user-images.githubusercontent.com/49918030/107312889-6e387b00-6acc-11eb-804c-b9079053dc79.png)
![image](https://user-images.githubusercontent.com/49918030/107312916-7abcd380-6acc-11eb-9170-912610e0d5c6.png)
希望能尽快更新一下支持新系统的问题，也感谢付出～

**Comment by SpaceGrey** [comment]
哇我碰到了一某一样的问题

**Reply by bosslay** [reply]
same +1

**Comment by Mcc-288847** [comment]
一样的问题

**Comment by ply0011** [comment]
same same same

**Comment by fanboynet** [comment]
前面好像有提过，重新签名下能暂时正常。
codesign --sign - --force --deep /Applications/WeChat.app
好像得安装xcode
